cloudamqp/lavinmq
# Dont write all binds to defs

### WHAT is this pull request doing?
definitions.amqp can grow indefinitely because we write all bind and unbind frames to it. This will add guard against that behavior.

### HOW can this pull request be tested?
Add the same binding multiple times and verify it's not written to definitions.amqp. If log level is debug it will be logged if written to file.
